variants/github_actions_ci/workflows/deploy_to_ec2.yml

```diff
@@ -0,0 +1,72 @@
+name: deploy_to_ec2
+on:
+  workflow_call:
+    inputs:
+      environment:
+        type: string
+        required: true
+      assume_role_arn:
+        type: string
+        required: true
+      aws_region:
+        type: string
+        required: true
+      timeout_minutes:
+        type: number
+        required: false
+        default: 15
+      slack_icon:
+        type: string
+        required: false
+        default: https://github.com/ackama.png?size=48
+      slack_channel:
+        type: string
+        required: false
+        default:
```

Comment:
I wonder if we should set this to a channel that doesn't exist, as otherwise the message will get sent to the default for the webhook being used (which want to be drawn from an org secret) - effectively we want a "required if `slack_webhook` present" which defaulting to a non-existent channel would get us without any extra effort.